### PR TITLE
Security issue by curl

### PR DESCRIPTION
**Pull request by voku**
The problem is that there is an not carefully filtered url: "if (isset($url_parsed)) {" 

... so we can fetch e.g. "file:///etc/passwd" and everything else on the server :/

-> http://security.stackexchange.com/questions/95650/is-allowing-unfiltered-curl-request-from-a-website-a-vulnerability

-> https://books.google.de/books?id=tGoLBAAAQBAJ&lpg=PA470&dq=%22CURLOPT_URL%22%20hack%20%22file%3A%2F%2F%22&hl=de&pg=PA474#v=onepage&q=%22CURLOPT_URL%22%20hack%20%22file://%22&f=false

### CODE REVIEW

**Comment by thelfensdrfer** [comment]
Thanks for this fix!

File: src/Piwik.php [hunk 1]
```diff
@@ -44,8 +44,12 @@ class Piwik
 
 	private $_errors = array();
 
-	public $verifySsl = true;
-	public $redirects = 3;
+	public $verifySsl = false;
+
+  public $maxRedirects = 5;
```
Comment (thelfensdrfer) [inline src/Piwik.php]: Can you please use tabs for your changes?

File: src/Piwik.php [hunk 2]
```diff
@@ -44,8 +44,12 @@ class Piwik
...
+
+  // WARNING: not used anymore
```
Comment (thelfensdrfer) [inline src/Piwik.php]: Replace with:

```
/**
 * @deprecated
 */
```
?

File: src/Piwik.php (hunk 1 above)
Comment (voku) [inline src/Piwik.php]: fixed

**Pull request merged**